src/TorchSharp/Tensor/Tensor.cs

```diff
@@ -5533,13 +5533,16 @@ static public TensorIndex Slice(long? start = null, long? stop = null, long? ste
                 return new TensorIndex() { startIndexOrBoolOrSingle = start, step = step, stopIndex = stop, kind = Kind.Slice };
             }
 
+            static public TensorIndex Slice((int? start, int? end) range) => TensorIndex.Slice((long?)range.start, (long?)range.end);
+
+#if !NETSTANDARD2_0
```

Comment:
Might make sense to add [IndexRange](https://www.nuget.org/packages/IndexRange/) dependency to `netstandard2.0` target.

Reply:
I don't think it is worth it to add a dependency on a third-party library here. doing that comes with other issues like not have enough control to address any security issues on time and also not-guarantee the compatibility moving forward.

Reply:
@tarekgh with 0.96 we don't need forward compatibility. I'd rather have no extra ifdefs for already niche 4.8 target.

Reply:
I don't think we're quite ready to designate 4.8 a 'niche' target yet.